### DISCUSSION TITLE
Questions and answers about nitda-blockchain-scholarship

### DISCUSSION THREAD

**Discussion by Habuchajii** [community/community#41524] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

imformation about nitda/blockchain/scholarship

**Comment by queenofcorgis** [comment]
Hi @Habuchajii ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as https://nitda.gov.ng/. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by RitaCummings** [comment]
oops, I got the same issue while entering this **_[crypto, bitcoin, and blockchain blog for newbies](https://brlansclub.cm/)_**.
